extra.bib

```diff
@@ -78,3 +78,21 @@ @Online{wzvgbayern
   url  =  {https://www.stmwk.bayern.de/download/13009_Grundsaetze_Befristungen_Fassung_2019_mErgDKuSG.pdf},
   urldate = {2025-05-16}
 }
+
+@misc{goth2025_WSPad,
+  author       = {Goth, Florian and
+                  Chourdakis, Gerasimos and
+                  Hagdorn, Magnus and
+                  Thiele, Jan Philipp and
+                  Cohen, Jeremy and
+                  Martone, Michele and
+                  Hodges, Toby and
+                  Barrios Sazo, Maria Guadalupe},
+  title        = {de-RSE25 BoF: The End of RSEng? Challenges and Risks for RSEng},
```

Comment:
Should we have something like "Notes from" or otherwise the word "workshop" or similar in the title? From the references list, it should be clearer that this is data/notes and not something peer-reviewed.